src/util/hash.ts

```diff
@@ -0,0 +1,10 @@
+import * as crypto from 'crypto';
+
+// import * as dotenv from "dotenv";
+// dotenv.config();
+
+const salt = process.env.SALT;
+
+export const hashSha512 : Function = (password : String) : String => {
+    return crypto.createHash('sha512').update(password + salt).digest('base64');
```

Comment:
salt를 추가할 때에는 createHash 함수가 아닌 createHmac 함수를 사용하면 돼요.
```suggestion
    return crypto.createHmac('sha512', salt).update(password).digest('base64');
```